ml_annotators/ml_annotators.ipynb

```diff
@@ -115,7 +125,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You'll see that there is now a table of coordinates and also that there is now a [PointDraw tool](../reference/streams/bokeh/PointDraw.ipynb) in the toolbar:<img src=\"https://bokeh.pydata.org/en/latest/_images/PointDraw.png\">\n",
+    "You'll see that there is now a table of coordinates and also that there is now a [PointDraw tool](https://holoviews.org/reference/streams/bokeh/PointDraw.html) in the toolbar:<img src=\"https://bokeh.pydata.org/en/latest/_images/PointDraw.png\">\n",
```

Comment:
I'm curious why the .ipynb links have been replaced with .html links in this file. The .ipynb ones are meant to turn into .html on the website, but should normally be .ipynb links in the notebooks so that people can navigate between the notebooks without reaching out to the external internet.

Reply:
Since the .ipynb files don't exist in the examples repo, I switched them to .html.